contexts/network-settings.tsx

```diff
@@ -209,7 +209,7 @@ export const NetworkSettingsProvider = ({ children }) => {
       validator: async (value: string) => {
         let validated = undefined;
   
-        if (value.trim() !== "")
+        if (value.trim() !== "" && !isSetup)
```

Comment:
this condition was removed on development, rebase or merge dev into your branch to make sure you have the latest changes :)